.github/workflows/push-to-master.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           # for more info see https://github.com/conda-incubator/setup-miniconda/issues/264
           use-only-tar-bz2: false
       - name: install Ray nightly build
-        run: pip install https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-2.0.0.dev0-cp38-cp38-manylinux2014_x86_64.whl
+        run: pip install https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-3.0.0.dev0-cp38-cp38-manylinux2014_x86_64.whl
```

Comment:
Have you tested it locally? (the installation itself, otherwise we won’t know until we merge it)

Reply:
Yep, I ran all the tests in this job and they passed.

Reply:
Good, thanks!